RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGeneratorKernels.cu

```diff
@@ -591,9 +745,22 @@ void CAHitQuadrupletGeneratorKernels::buildDoublets(HitsOnCPU const &hh, cuda::s
 
   device_theCells_ = cs->make_device_unique<GPUCACell[]>(CAConstants::maxNumberOfDoublets(), stream);
 
+#ifdef GPU_DEBUG
+  cudaDeviceSynchronize();
+  cudaCheck(cudaGetLastError());
+#endif
+
   if (0 == nhits)
     return;  // protect against empty events
 
+  // FIXME avoid magic numbers
```

Comment:
Agreed :-)
Do you already have a patch for this in the following PRs ?

Reply:
nope